### PR TITLE
use latex font

### PR DESCRIPTION
**Pull request by dragonyanglong**
Hi @sbillinge , I updated the latex font issue you mentioned. BTW, how can we feel the difference? I screenshot the old and new figures below:

OLD:
plot

NEW:
plot1

### CODE REVIEW

**Comment by dragonyanglong** [comment]
Hi @sbillinge , how about this? the same A as in Amstrong.

image

**Comment by dragonyanglong** [comment]
Or this: same A as in Amstrong as well.

image

**Comment by sbillinge** [comment]
> 
> 
> Hi @sbillinge , how about this? the same A as in Amstrong.
> 
> image

I think this one looks right.  In the second case the A looks as if it is bolder than the other letters and numbers.  Maybe it is a different font?  Bottom line, I think this is it!

**Comment by dragonyanglong** [comment]
Thanks Simon for double checking it. I pushed. Please review and merge.

File: bg_mpl_stylesheet/bg_mpl_stylesheet [hunk 1]
```diff
@@ -203,7 +203,7 @@ font.family          : arial
 ## small, medium, large, x-large, xx-large, larger, or smaller
 font.size            : 15
 #font.serif          : DejaVu Serif, Bitstream Vera Serif, Computer Modern Roman, New Century Schoolbook, Century Schoolbook L, Utopia, ITC Bookman, Bookman, Nimbus Roman No9 L, Times New Roman, Times, Palatino, Charter, serif
-#font.sans-serif     : DejaVu Sans, Bitstream Vera Sans, Computer Modern Sans Serif, Lucida Grande, Verdana, Geneva, Lucid, Arial, Helvetica, Avant Garde, sans-serif
+font.sans-serif      : cm
```
Comment (sbillinge) [inline bg_mpl_stylesheet/bg_mpl_stylesheet:206]: I doubt this is the right way to do it.  We want to keep this somewhat generic font family in place, but select from it.  It looks as if below you select 'cm' from the font family sans-serif, so you just need to add cm to the family. Does that work?

**Review by sbillinge — COMMENTED** [review]
one last question.

**Comment by dragonyanglong** [comment]
@sbillinge please merge.

**Pull request merged**